src/autopas/molecularDynamics/LJFunctor.h

```diff
@@ -937,13 +855,10 @@ class LJFunctor
     const size_t neighborListSize = neighborList.size();
     const size_t *const __restrict__ neighborListPtr = neighborList.data();
 
-    // checks whether particle 1 is in the domain box, unused if _duplicatedCalculations is false!
-    SoAFloatPrecision inbox1Mul = 0.;
-    if (duplicatedCalculations) {  // only for duplicated calculations we need this value
-      inbox1Mul = ownedPtr[indexFirst];
-      if (newton3) {
-        inbox1Mul *= 0.5;
-      }
+    // checks whether particle 1 is in the domain box, unused if calculateGlobals is false!
```

Comment:
```suggestion
    // checks whether particle I is in the domain box. Unused if calculateGlobals is false!
```
THe second part seems to be not true anymore.